beacon-chain/sync/rpc_beacon_blocks_by_range.go

```diff
@@ -21,11 +21,6 @@ import (
 func (s *Service) beaconBlocksByRangeRPCHandler(ctx context.Context, msg interface{}, stream libp2pcore.Stream) error {
 	ctx, span := trace.StartSpan(ctx, "sync.BeaconBlocksByRangeHandler")
 	defer span.End()
-	defer func() {
```

Comment:
I'm now explicitly closing on success. If we don't close the stream, the main RPC handler will reset it.